src/Parse/ParseQuery.php

```diff
@@ -623,6 +623,26 @@ public function withinGeoBox($key, $southwest, $northeast)
     }
 
     /**
+     * Add a constraint to the query that requires a particular key's
+     * coordinates be contained within a given polygon
+     *
+     * @param string        $key       The key of the ParseGeoPoint
+     * @param array         $points    Array of ParseGeoPoint
+     *
+     * @return ParseQuery Returns this query, so you can chain this call.
+     */
+    public function withinPolygon($key, $points)
+    {
+        $this->addCondition(
+            $key,
+            '$geoWithin',
+            ['$polygon' => $points]
+        );
+
+        return $this;
+    }
+
```

Comment:
@dplewis nice catch! Although it is helpful to check against multiple db types I would say such checks are more appropriate when running parse-server's test suite. That issue _should_ have been caught during routine testing.

Basically when I run tests against this I'm making a certain degree of assumptions, first and foremost being that the database integrations are properly tested for compatibility before rolling out. Granted this is only an assumption, so issues may come to light from time to time.